app/views/clients/show.html.erb

```diff
@@ -80,6 +80,9 @@
   <strong>local_address:</strong>
   <%= @client.local_address %><br>
 
+  <strong>local_city:</strong>
+  <%= @client.local_city %><br>
```

Comment:
If you fill out the form and try to submit it, this line causes an error. This is because the `local_city` field doesn't exist in the clients table. To fix this, you'll need to create a migration. We might want to pair on this, and think about how to do it carefully.